pkg/kubelet/cm/memorymanager/state/state_checkpoint_test.go

```diff
@@ -284,180 +301,80 @@ func TestCheckpointStateStore(t *testing.T) {
 			// ensure there is no previous checkpoint
 			cpm.RemoveCheckpoint(testingCheckpoint)
 
-			cs1, err := NewCheckpointState(testingDir, testingCheckpoint, nil)
+			state, err := NewCheckpointState(testingDir, testingCheckpoint, nil)
 			if err != nil {
 				t.Fatalf("could not create testing checkpointState instance: %v", err)
 			}
+			state.SetMachineState(tc.machineState)
```

Comment:
Do we need to set machine state? I am curious because next tests only check memory blocks.
Also, do we need to define machine state (above) if we only check memory blocks (below)?

Reply:
the good point will check the machine state as well